.evergreen/releases.yml

```diff
@@ -137,19 +138,38 @@ functions:
         key_id: ${PAPERTRAIL_KEY_ID}
         secret_key: ${PAPERTRAIL_SECRET_KEY}
         product: ${PAPERTRAIL_PRODUCT}
-        version: ${GIT_TAG}
+        version: ${CRATE_VERSION}
         filenames:
-          - src/target/package/mongodb-*.crate
+          - src/target/package/mongodb-${CRATE_VERSION}.crate
+
+  "sign release":
+    - command: subprocess.exec
+      params:
+        working_dir: "src"
+        include_expansions_in_env:
+          - ARTIFACTORY_USERNAME
+          - ARTIFACTORY_PASSWORD
+          - GARASIGN_USERNAME
+          - GARASIGN_PASSWORD
+          - CRATE_VERSION
+        binary: bash
+        args:
+          - .evergreen/release-sign.sh
+
+  "save signature":
```

Comment:
This is a hack; once DEVPROD-7047 is done this can just be an `s3.put` of the signature file specifically and then we can include it in github releases / link to the URL / whatever.